### DISCUSSION TITLE
Enable Single View as a source collection for Mongo-DB Views on console

### DISCUSSION THREAD

**Discussion by giuliowaitforitdavide** [mia-platform/community#95] [Ideas]
Hi to all,
I would like to use MongoDB Views to aggregate some data I have in my Single Views, but right now the console does not support Single Views as source collection of a MongoDB View. Would it be a cool idea to add this support?

**Comment by TheRusCompass** [comment]
Hello! Thank you for your suggestion! With the new v10.1.0 release of the console you will have the possibility to create MongoDB Views for both Projections and Single Views!
